php-frontend/src/main/java/org/sonar/php/parser/PHPLexicalGrammar.java

```diff
@@ -386,8 +386,8 @@ private static void keywords(LexerlessGrammarBuilder b) {
 
     b.rule(KEYWORDS).is(SPACING,
       b.firstOf(
-        PHPKeyword.getKeywordValues()[0],
-        PHPKeyword.getKeywordValues()[1],
+        b.regexp("(?i)" + PHPKeyword.getKeywordValues()[0]),
+        b.regexp("(?i)" + PHPKeyword.getKeywordValues()[1]),
```

Comment:
Maybe we can extract some logic in a dedicated method like:
```
keywordRegexp(String keywordValue) {
  return b.regexp("(?i)" + keywordValue);
}
```
I think we would use this method on lines: 376, 380, 382, 389, and 390).